src/Components/Books.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Books({ title, author }) {
+  return (
+    <div className="main">
+      <div>
+        <h1>{title}</h1>
+        <p>
+          By
+          {author}
+        </p>
```

Comment:
- [ ] You have done great implementing some of the requirements from the previous review, :+1: but there are still a few requirements you need to meet before moving to the next net milestone. :+1: :100: 
- [ ] As the previous reviewer was trying to point out to you, here's what you need.
- - First is a `Books.js` page component. here you will import the `<BookList>` and ``<NewBookForm>`` component.
- - - Screenshot:
 
![Screenshot from 2023-08-16 17-19-40](https://github.com/Wils-FOURR-speed/bookstore/assets/107632780/d55c2496-57c7-4234-9585-1337e6f4bbb1)

- - The second is a `BookList.js` component. here you can have an array of books in the state that you can loop through and render each book using the `Book.js` component.
- - - Screenshot:
 
![Screenshot from 2023-08-16 17-19-40](https://github.com/Wils-FOURR-speed/bookstore/assets/107632780/b36c3f7a-44ff-496b-a1e1-87454582f0d5)

- - Then finally a `Book.js` component renders the book name and author with the buttons `edit, delete`.
- - - Screenshot:

![Screenshot from 2023-08-16 17-28-44](https://github.com/Wils-FOURR-speed/bookstore/assets/107632780/84b8f315-7494-4890-a523-2508fff94115)

I'll suggest you update your code accordingly. :+1: :+1:

Reply:
@V-Blaze please I'm confused. You're talking about Books.js everywhere
Could you be more specific?